reference_designs/ipu-xrt/matrix_vector_multiplication/aie2.py

```diff
@@ -103,9 +103,9 @@ def device_body():
                     2,
                     memRef_A_ty,
                     [
-                        (k_in_i32s, 1),
-                        (m, k_in_i32s),
-                        (1, 1),
+                        (k_in_i32s, 2),
```

Comment:
Why are we using k_in_i32s here? Does more of this PR need to be absorbed in the objFIFO?

Reply:
I was hoping to see here k and not k_in_i32s.